test/performance/odp_ipsecfwd.c

```diff
@@ -374,18 +375,10 @@ static void print_usage(void)
 	       "            }\n"
 	       "        );\n"
 	       "\n"
-	       "Usage: %s [options]\n"
+	       "Mandatory [OPTIONS]:\n"
```

Comment:
Maybe "Mandatory options" instead, i.e. without the brackets. And the same for the optional options.

Reply:
Without brackets in v2. Still uppercase as that seems to the style in almost all of our testers.